app/src/androidTest/java/com/xmartlabs/template/common/ImmediateNewThreadScheduler.kt

```diff
@@ -20,13 +20,9 @@ class ImmediateNewThreadScheduler : Scheduler() {
       return schedule(SleepingAction(this, execTime, unit) { run.run() })
     }
 
-    override fun dispose() {
-      innerSubscription.cancel()
-    }
+    override fun dispose() = innerSubscription.cancel()
 
-    override fun isDisposed(): Boolean {
-      return innerSubscription.isCancelled
-    }
+    override fun isDisposed() = innerSubscription.isCancelled
```

Comment:
Yes, it's pretty much the same, but the type is inferred, which is nice. It's called a `expression function`